contrib/builder/rpm/armhf/README.md

```diff
@@ -0,0 +1,5 @@
+# `dockercore/builder-rpm`
```

Comment:
I think we stopped adding READMEs for each architecture, as they are all the same.